portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,14 +65,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String regex = "\\s+$";
         long timestamp = System.currentTimeMillis();
         comment = comment.replaceAll(regex, "");
-        posts.add(text);
 
         Entity commentEntity = new Entity("Comment");
-        commentEntity.setProperty("comment", text);
+        commentEntity.setProperty("comment", comment);
         commentEntity.setProperty("timestamp", timestamp);
 
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         datastore.put(commentEntity);
+        System.out.println(commentEntity);
```

Comment:
nit: this comment might not be so useful now. If you want to keep it in, consider adding context like you did on line 54.